socketshark/__init__.py

```diff
@@ -97,8 +97,9 @@ def signal_shutdown(self):
         redis_receiver = Receiver(loop=asyncio.get_event_loop())
         redis_settings = self.config['REDIS']
         try:
-            self.redis = await aioredis.create_redis((
-                redis_settings['host'], redis_settings['port']))
+            self.redis = await aioredis.create_redis(
+                (redis_settings['host'], redis_settings['port']),
+                db=redis_settings['db'])
```

Comment:
Please fall back to 0 if it isn't in the config, otherwise this is a backwards-incompatible change.